src/VCR/Storage/AbstractStorage.php

```diff
@@ -54,10 +54,12 @@
      * If the cassetteName contains PATH_SEPARATORs, subfolders of the
      * cassettePath are autocreated when not existing.
      *
-     * @param string $cassettePath Path to the cassette directory.
-     * @param string $cassetteName Path to the cassette file, relative to the path.
+     * @param string  $cassettePath   Path to the cassette directory.
+     * @param string  $cassetteName   Path to the cassette file, relative to the path.
+     * @param string  $defaultContent Default data for this cassette if its not existing
+     * @param boolean $writeable      Whether the file must be writeable
      */
-    public function __construct($cassettePath, $cassetteName, $defaultContent = '[]')
+    public function __construct($cassettePath, $cassetteName, $defaultContent = '[]', $writeable = true)
```

Comment:
maybe the better idea, especially as the yaml constructor follows a different pattern (causing the failures in travis).

this delays reporting the error to when writing is actually attempted but whatever, its just testing so you are supposed to look at the output. i will change to that